examples/jsm/loaders/OBJLoader.js

```diff
@@ -515,7 +515,8 @@ class OBJLoader extends Loader {
 			const lineFirstChar = line.charAt( 0 );
 
 			// @todo invoke passed in handler if any
-			if ( lineFirstChar === '#' ) continue;
+			const thisLineIsAComment = lineFirstChar === '#';
+			if ( thisLineIsAComment ) continue;
```

Comment:
Instead of changing the code, please just add a comment:
```js
if ( lineFirstChar === '#' ) continue; // skip comments
```

Reply:
Okay, I made the change you suggested.